web-ng/src/app/components/project-header/project-header.component.ts

```diff
@@ -31,6 +33,7 @@ export class ProjectHeaderComponent implements OnInit, OnDestroy {
   lang: string;
   title: string;
   projectId!: string;
+  offlineBaseMapSources?: OfflineBaseMapSource[];
```

Comment:
Any reason these are being passed from the header, instead of just being set in the `createProject` method?